### ISSUE TITLE
`iroh p2p lookup`: Hard to distinguish relay IPs from actual IP

### ISSUE THREAD

**Issue by Winterhuman** [n0-computer/iroh#470]
`iroh p2p lookup` prints all observed and listening addresses, which includes both your actual IP and the IPs of relays, however, it's not obvious which are which.

Ideally, the output for relays would show `/ip4/1.2.3.4/tcp/567/p2p-circuit` to make them distinct

**Comment by Winterhuman** [comment]
Oh, actually this seems to be different problem, doing `ipfs id PeerID` for the Iroh node reveals it's advertising relay nodes as your own node, it's showing:

```
/ip4/IP-of-Relay/tcp/4444/p2p/PeerID-of-your-node
```

**Comment by dignifiedquire** [comment]
That seems to be what should be advertised when relay is enabled? Or what am I missing? cc @mxinden

**Comment by Winterhuman** [comment]
@dignifiedquire Relay multiaddresses should look like:

```
/ip4/"Relay IP"/tcp/1234/p2p/"Relay PeerID"/p2p-circuit/p2p/"Node PeerID"
```

The multiaddresses I show are implying that the node has the IP address, not a relay that it's accessible through

**Comment by mxinden** [comment]
> Relay multiaddresses should look like this:
> 
> ```
> /ip4/"Relay IP"/tcp/1234/p2p/"Relay PeerID"/p2p-circuit/p2p/"Node PeerID"
> ```

Yes. A node should advertise the above as their relayed address. @Winterhuman have you been able to track down where the issue is?

**Comment by dignifiedquire** [comment]
Ah yes. I really wonder where this comes from, as we don't do anything with these addresses on the iroh side other than displaying them asfaik.
